visualdl/server/lib.py

```diff
@@ -73,7 +73,12 @@ def get_scalar(storage, mode, tag, num_records=300):
 
                 for index in range(len(data)):
                     data_reservoir.add_item(mode=mode, tag=tag, item=data[index])
-        return data_reservoir.get_items(mode, tag)
+        results = data_reservoir.get_items(mode, tag)
+        # TODO(Superjomn) some bug here, sometimes there are zero here.
```

Comment:
Is this comments nessasary?

Reply:
This error exists in version 1.3.0. I took a plan to avoid it, but this problem still need to be resolved from the underlying architecture.

Reply:
OK, add this TODO in our plans.